src/cyrillux.js

```diff
@@ -16,22 +16,25 @@ const cyrillicTrans = 'CYRILLIC <TRANS>';
 const sfs4900Trans = 'SFS4900 <TRANS>';
 
 export default function (config = {}) {
+  // console.log(`CONFIG=${JSON.stringify(config)}`); // eslint-disable-line no-console
 
   return {
     description: 'Cyrillux functionality: convert original field to latinitsa (ISO-9) and add 880s for original cyrillic and latinitsa (SFS-4900)',
     validate, fix
   };
 
   function preprocessConfig() {
+    config.retainCyrillic = typeof config.retainCyrillic === 'undefined' ? true : config.retainCyrillic; // eslint-disable-line functional/immutable-data
```

Comment:
`config.retainCyrillic = config.retainCyrillic === undefined ? true : config.retainCyrillic;`

Reply:
This function could return object like
```js
return {
 retainCyrillic: config.retainCyrillic === undefined ? true : config.retainCyrillic, 
...
};
```